packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -386,8 +424,12 @@ export class ResourceService {
       throw new Error(INVALID_RESOURCE_ID);
     }
 
-    if (resource.resourceType === EnumResourceType.ProjectConfiguration) {
-      throw new Error(INVALID_DELETE_PROJECT_CONFIGURATION);
+    switch (resource.resourceType) {
+      case EnumResourceType.ProjectConfiguration:
+        throw new Error(INVALID_DELETE_PROJECT_CONFIGURATION);
```

Comment:
I think @MichaelSolati worked on project deletion, which, if I'm not mistaken, is basically deleted to project configuration.

Reply:
Not changes have been made here apart from a minor refactoring.